docs/Doxyfile

```diff
@@ -993,7 +993,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                =
+EXCLUDE                = lib/librom_wrap.cxx
```

Comment:
We don't want to generate documentation for this SWIG file, so let's exclude it.